ui/app/pages/first-time-flow/seed-phrase/seed-phrase.component.js

```diff
@@ -39,12 +41,15 @@ export default class SeedPhrase extends PureComponent {
   }
 
   render() {
-    const { seedPhrase } = this.props;
+    const { seedPhrase, history } = this.props;
     const { verifiedSeedPhrase } = this.state;
+    const pathname = history?.location?.pathname;
+    const introClass =
+      pathname === INITIALIZE_SEED_PHRASE_INTRO_ROUTE ? 'intro' : '';
 
     return (
       <DragDropContextProvider backend={HTML5Backend}>
-        <div className="first-time-flow__wrapper">
+        <div className={`first-time-flow__wrapper ${introClass}`}>
```

Comment:
When we modify the rest of the onboarding flow to follow the new designs, we can just bake rules directly in to `first-time-flow__wrapper`. Presently, it doesn't play so nicely with the new video screen